app/components/nft-web-view/nft-web-view.tsx

```diff
@@ -15,14 +35,36 @@ export function NFTWebView(props: WebViewProps) {
     return true
   }
 
+  const handleBackButtonPress = () => {
+    webViewRef.current?.goBack()
+  }
+
+  const handleHomeButtonPress = () => {
+    // HACK: Switching key to force go to the initial page
+    setWebViewKey((webViewKey + 1) % 2)
+  }
+
+  const handleRefreshButtonPress = () => {
+    webViewRef.current?.reload()
+  }
+
   return (
-    <WebView
-      {...props}
-      sharedCookiesEnabled={true}
-      decelerationRate={0.998}
-      // TODO: remove HACK after applicationNameForUserAgent type is fixed
-      {...{ applicationNameForUserAgent: COMMON_API_CONFIG.userAgent }}
-      onShouldStartLoadWithRequest={handleShouldStartLoadWithRequest}
-    />
+    <View style={style}>
+      <WebView
+        key={`${webViewKey}}`}
+        ref={webViewRef}
+        {...props}
+        sharedCookiesEnabled={true}
+        decelerationRate={0.998}
+        // TODO: remove HACK after applicationNameForUserAgent type is fixed
+        {...{ applicationNameForUserAgent: COMMON_API_CONFIG.userAgent }}
+        onShouldStartLoadWithRequest={handleShouldStartLoadWithRequest}
+      />
+      <ControlBar>
+        <Button icon="back" size="tiny" preset="plain" onPress={handleBackButtonPress} />
```

Comment:
Can we handle android back button event to call this also?

Reply:
Added